apps/arena_load_test/lib/arena_load_test/socket_supervisor.ex

```diff
@@ -31,9 +32,17 @@ defmodule ArenaLoadTest.SocketSupervisor do
 
   # Creates `num_clients` clients to join a game
   def spawn_players(num_clients) do
-    for i <- 1..num_clients do
-      {:ok, _pid} = add_new_client(i)
+    case :ets.whereis(:clients) do
+      :undefined -> :ets.new(:clients, [:set, :named_table, :public])
+      _table_exists_already -> nil
     end
+
+    Enum.each(1..num_clients, fn client_number ->
+      Logger.info("Iteration: #{client_number}")
+      {:ok, _pid} = add_new_client(client_number)
+      true = :ets.insert(:clients, {client_number, "1"})
```

Comment:
you could also save the pid in the table, just to replace the magic 1

Reply:
You're right. However, I don't really need any value, I just want to store client_id's and know how many clients are in the table.